### DISCUSSION TITLE
Is the package already compatible with iOS15 & Xcode 13 Beta?

### DISCUSSION THREAD

**Discussion by lockinmarv** [firebase/firebase-ios-sdk#8255] [Q&A] [answered]
Hey! 
I'm getting warnings like `Invalid exclude` for the `DerivedData` folder several times when I've installed the package in Xcode 13 Beta. Is the package already compatible with it and can I ignore these warnings?
I also cleared the `DerivedData` folder but the warnings still appear.

**Comment by paulb777** [comment, marked as answer]
Hey @MarvinRudolph, thanks for the report!  We have fixes in progress for this Xcode 13 and hope to have package updates out soon. In the meantime, it is safe to ignore the warnings.

**Reply by lockinmarv** [reply]
Hey, thanks for your fast answer! Great to hear. :)

**Reply by ryanwilson** [reply]
boringSSL-SwiftPM, GoogleUtilities, and abseil-cpp-SwiftPM repos were updated so those warnings should now go away after doing an update. The warnings in this repo will be fixed in the next release.
